src/elements/content-sidebar/versions/VersionsSidebarContainer.js

```diff
@@ -132,8 +133,7 @@ class VersionsSidebarContainer extends React.Component<Props, State> {
 
         return this.api
             .restoreVersion(this.findVersion(versionId))
-            .then(this.api.fetchData)
-            .then(this.handleFetchSuccess)
+            .then(this.mergeResponse)
```

Comment:
Can we introduce a `handleRestoreSuccess` helper that calls `mergeVersions` and then sets state?